docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,151 @@
+# Release 405 (23 Nov 2022)
+
+```{note}
+Where's release 404? Not found.
+```
+
+## General
+
+* Improve performance of queries that compare `date` columns with
+  `timestamp with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization of `INSERT` queries into
+  partitioned and bucketed tables. ({issue}`14718`)
+* Fix overflows of `TIME` and `TIME WITH TIME ZONE` types by rounding to zero. ({issue}`15138`)
+* Fix incorrect query results when an un-partitioned `WINDOW` operator is
+  followed by partitioned and ordered `WINDOW` operators. {(issue)`15203`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Fix some `system.metadata` tables improperly showing the names of catalogs
+  which the user cannot access. ({issue}`14000`)
+* Fix `USE` statement improperly showing the names of catalogs and schemas which
+  the user cannot access. ({issue}`14208`)
+
+## BigQuery connector
+
+* Add support for retrying when creating a BigQuery read session. ({issue}`15013`)
+* Fix queries incorrectly executing with the `project-id` specified in the
+  credentials instead of the project ID specified in the `bigquery.project-id`
+  catalog property. ({issue}`14083`)
+
+## ClickHouse connector
+
+* Add context comments to `SELECT` queries. ({issue}`14500`)
```

Comment:
It would be better to explain "context comments" in my opinion.

Reply:
Is there something specific you'd propose?

Reply:
What is a "context comment"?